SmellDetector/SmellDetector.csproj

```diff
@@ -2,4 +2,12 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
   </PropertyGroup>
+  
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
+    <PackageReference Include="RabbitMQ.Client" Version="6.2.1" />
+    <PackageReference Include="Shouldly" Version="3.0.2" />
```

Comment:
A non-test project should not reference Shouldly. The SmellDetectorTests project should(ly).